test/e2e/crd.go

```diff
@@ -43,9 +46,69 @@ import (
 	. "github.com/onsi/ginkgo"
 )
 
+const crdGroup string = "example.com"
```

Comment:
I would prefer that you follow the example I set in #673:

https://github.com/kubernetes-sigs/federation-v2/pull/673/files#diff-39161408e007fa958c893de39a5332d3R85

I don't think it is desirable for this crd testing to validate kube's rbac implementation, and changing the group and version to something the test will have permission for allows the test to run with minimal effort.

Reply:
ok. I will rebase it.